src/Utility/common.js

```diff
@@ -56,6 +56,23 @@ export function hash(s) {
     );
 }
 
+export function almostUniqueId(length = 9) {
+    const d = new Date();
+    return (
+        ('' + d.getUTCFullYear()).slice(-2) +
+        ('0' + (d.getUTCMonth() + 1)).slice(-2) +
+        ('0' + d.getUTCDate()).slice(-2) +
+        Math.random()
+            .toString(36)
+            .substr(2, length)
+            .toUpperCase()
+    );
+}
+
+export function renderMoney(amount, currency = '') {
+    return Number(amount).toLocaleString(LOCALE, { minimumFractionDigits: 2 }) + (currency ? ' ' + currency : '');
```

Comment:
Apparently this shows 3 digits which is one too many…

Reply:
And the QR Codes seem to be broken in print.

Reply:
> Apparently this shows 3 digits which is one too many…

Where? Looks fine for me.

Reply:
> And the QR Codes seem to be broken in print.

Fixed.

Reply:
> Where? Looks fine for me.

<img width="811" alt="Screen Shot 2019-12-10 at 14 22 00" src="https://user-images.githubusercontent.com/4049135/70533101-43f9cf00-1b50-11ea-8764-24cf1cdf5a59.png">

Reply:
155d14a